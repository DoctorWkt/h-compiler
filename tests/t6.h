char a[]= {1,2,7,4,5,6,7,8};
void main()
{
  int x;
  int y;
  for (x=0; x != 8; x++) {
    y= a[x];
    printn(y);
  }
}

