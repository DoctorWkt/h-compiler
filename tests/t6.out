000001
000002
000007
000004
000005
000006
000007
000010
